cmd/controller/controller.go

```diff
@@ -237,7 +239,7 @@ func (p *lvmProvisioner) createProvisionerPod(va volumeAction) (err error) {
 	privileged := true
 	provisionerPod := &v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: string(va.action) + "-" + va.name,
+			Name: p.vgName + string(va.action) + "-" + va.name,
```

Comment:
yep,I know but i think that it will be more correct to create pod with vgName prefix and more convenient to see pod with vgName when you test or debug smt

Reply:
I would suggest adding a hyphen for better readability:

`p.vgName + "-" + string(va.action) + "-" + va.name
`
now: `create-pvc-b4edff12-a202-49b1-9625-c68406459f21`
then: `csi-lvm-create-pvc-b4edff12-a202-49b1-9625-c68406459f21` vs `csi-lvmcreate-pvc-b4edff12-a202-49b1-9625-c68406459f21` (still less then 63 characters max podname length)

Reply:
no, this is too verbose, otherwise all kind of configuration options end in the podname

Reply:
Ok, you win, it's not so critical.